app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectsListActivity.kt

```diff
@@ -29,8 +29,15 @@ class ProjectsListActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_projects_list)
+
+        val fromLink = intent.getBooleanExtra("fromLink", false)
+        var projectId = ""
+        if (fromLink) {
+            projectId = intent.getStringExtra("projectId") ?: ""
+        }
```

Comment:
Same as what I said in `MainActivity: line 64-65`

Reply:
done